### PR TITLE
New package: Teclib'.GLPIAgent1.7.2 version 1.7.2

### PR DESCRIPTION
**Pull request by MaxKoerck**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146786)

### CODE REVIEW

**Comment by stephengillie** [comment]
This package appears to be scripted at first glance, but the Windows Batch Files in the application root folder call a compiled EXE a few folders deeper in the directory structure. So I think it's okay.

**Review by stephengillie — APPROVED** [review]

File: manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Teclib'.GLPIAgent1.7.2
```
Comment (Trenly) [inline manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.installer.yaml:4]:
```suggestion
PackageIdentifier: Teclib.GLPIAgent
```
Versions don't need to be part of the identifier

File: manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Teclib'.GLPIAgent1.7.2
```
Comment (Trenly) [inline manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Teclib.GLPIAgent
```

File: manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Teclib'.GLPIAgent1.7.2
+PackageVersion: 1.7.2
+PackageLocale: en-US
+Publisher: Teclib'
+PackageName: GLPI Agent 1.7.2
+License: MIT
+ShortDescription: GLPI Agent 1.7.2
```
Comment (Trenly) [inline manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.locale.en-US.yaml:10]: Please provide a better description of what this package does or is used for

File: manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Teclib'.GLPIAgent1.7.2
```
Comment (Trenly) [inline manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.yaml:4]:
```suggestion
PackageIdentifier: Teclib.GLPIAgent
```

File: manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Teclib'.GLPIAgent1.7.2
+PackageVersion: 1.7.2
+PackageLocale: en-US
+Publisher: Teclib'
+PackageName: GLPI Agent 1.7.2
```
Comment (Trenly) [inline manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.locale.en-US.yaml:8]:
```suggestion
PackageName: GLPI Agent
```
Having the version here will cause matching errors in the future

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**